frontend/src/components/explorer/gemBrowser/MapsAvailable.vue

```diff
@@ -33,12 +35,14 @@
               </div>
             </td>
             <td>
-              <router-link :to="{ name: 'viewer',
-                                  params: { model: model.short_name, map_id: component.id,
-                                            reload: true },
-                                  query: { dim: '3d', search: viewerSelectedID, sel: viewerSelectedID } }">
-                3D
-                </router-link>
+              <div class="button is-outlined is-small link-button">
```

Comment:
Since I suggested this, I realise it would have helped to provide a clear example - I imagined the `white` button from [here](https://bulma.io/documentation/elements/button/#colors).

Reply:
![image](https://user-images.githubusercontent.com/32480198/122777363-42b07300-d2ac-11eb-90d7-e94192b9cd85.png)

You would prefer something like this instead of the current more buttonlike layout?

Reply:
Yeah, I personally liked that it was less crowded, but still felt the need for a button-like behaviour when hovering. What do you think?

Reply:
I personally like the buttons and I think it's a good balance of being clear and having a good "weight". They also look nice and consistent alongside the select if there is one (cytosol). The white button would not directly help users with touch devices.

Reply:
Good points @e0.

@inghylt it's your call!

Reply:
To have the buttons look good together with select for components with cytosol, it looks better to keep things as they are